src/less/tooltip/ds6/tooltip.less

```diff
@@ -0,0 +1,404 @@
+@import "../../less/ds6/variables.less";
+@import "../../less/ds6/mixins.less";
+
+@import "tooltip-base.less";
+
+/* Default style */
+.tooltip,
+.infotip,
+.tourtip {
+    display: inline-block;
+    position: relative;
+
+    &__window {
+        position: absolute;
+        z-index: 1;
+
+        &::after {
+            border-style: solid;
+            border-width: 8px;
+            content: "";
+            position: absolute;
+        }
+    }
+
+    &__body {
+        margin: 16px 16px 16px 16px;
+    }
+}
+
+.tooltip {
+    &:hover {
+        .tooltip__top-arrow,
+        .tooltip__top-leftarrow,
+        .tooltip__top-rightarrow,
+        .tooltip__bottom-arrow,
+        .tooltip__bottom-leftarrow,
+        .tooltip__bottom-rightarrow,
+        .tooltip__left-arrow,
+        .tooltip__left-toparrow,
+        .tooltip__left-bottomarrow,
+        .tooltip__right-arrow,
+        .tooltip__right-toparrow,
+        .tooltip__right-bottomarrow {
+            visibility: visible;
```

Comment:
@seangates Can you provide input here.
This is a negative pattern according to a11y guidelines.
https://ebay.gitbooks.io/mindpatterns/content/disclosure/tooltip.html

We should be rendering the tooltip as visible content, and then hiding it using javascript.  That means its not a pure CSS component, but I'm thinking that's the right way to go though.  We'll have to give an example of the JS needed to hide/show the tooltip.

Reply:
This is definitely an issue and has to be rethought.  The current demo is not accessible and we're teaching them an anti-pattern.  The tooltip isn't keyboard accessible.